File: 10041.cpp
/* a737: 10041 - Vito's family */
#include<iostream>
#include<algorithm>
using namespace std;

int main(){
    int n;
    cin>>n;
    for(int i=0;i<n;i++){
        int r;
        cin>>r;
        int R[r]={};
        for(int j=0;j<r;j++) cin>>R[j];
        sort(R,R+r);

        int half=r/2;
        int a=R[half], b=R[half-1];
        int tempa=0, tempb=0;

        for(int j=0;j<r;j++){
            tempa += abs(a-R[j]);
            tempb += abs(b-R[j]);
        }

        cout<<min(tempa,tempb)<<endl;

    }
    return 0;
}

File: 10041.py
'''a737: 10041 - Vito's family'''
import sys
n = input()

for i in range(int(n)):
    s = input()
    n = int(s.split(' ')[0])
    listt = s.split(' ')[1:]
    listt = sorted([int(k) for k in listt])
    
    half  = n//2
    a = listt[half]
    b = listt[half-1]

    tempa = 0
    tempb = 0
    for j in listt:
        tempa += abs(a-j)
        tempb += abs(b-j)

    print(min(tempa, tempb))
    